capstone/src/main/resources/static/event.js

```diff
@@ -0,0 +1,51 @@
+async function getEvents() {
+  const response = await fetch('get-all-events');
+  const jsonEvents = await response.json();
+  const eventList = document.getElementById('events-container');
+
+  jsonEvents.forEach(event => eventList.appendChild(createEvent(event)));
+}
+
+function createEvent(event) {
+  const listEventElement = document.createElement('li');
+  listEventElement.className = 'list-events';
+
+  // const organizationID = document.createElement('p');
+  // organizationID.innerText = event.organizationID;
+  // listEventElement.appendChild(organizationID);
+
+  const organizationName = document.createElement('p');
+  organizationName.innerText = event.organizationName;
+  listEventElement.appendChild(organizationName);
+
+  const eventTitle = document.createElement('p');
```

Comment:
These 3 lines repeat a lot. There is an opportunity here to create a helper method!